fuzzysearch-common/src/types.rs

```diff
@@ -54,6 +54,16 @@ pub struct SearchResult {
     pub searched_hash: Option<i64>,
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct E621Flags {
```

Comment:
I noticed that the some of the declarations in this file have `serde` serializer descriptions applied to them. Does this struct need one as well?